testsuite/integration/microprofile-tck/health/src/test/resources/arquillian.xml

```diff
@@ -7,17 +7,18 @@
 <arquillian xmlns="http://jboss.org/schema/arquillian" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
 
-    <container qualifier="jboss" default="true" >
+    <container qualifier="jboss" default="true">
         <configuration>
+            <property name="allowConnectingToRunningServer">true</property>
+            <property name="javaHome">${container.java.home}</property>
+            <property name="javaVmArguments">${microprofile.jvm.args} -Dmp.health.disable-default-procedures=true</property>
```

Comment:
Why is this needed?

Reply:
@bstansberry Good point, this is a rebase problem - as this was removed via https://issues.redhat.com/browse/WFLY-19147 and is provided through the pom here https://github.com/wildfly/wildfly/blob/35.0.0.Final/testsuite/integration/microprofile-tck/health/pom.xml#L95 instead. Fixed now.